src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -4285,15 +4297,7 @@ compiler.note.method.ir.dump=\
     code reflection enabled for method {0}.{1}\n\
     {2}
 
-# 0: symbol, 1: symbol, 2: string
```

Comment:
these "skip" diagnostics are a leftover from when we did not support all possible AST nodes. Since we should now support everything, I've removed this.